docs/vc-http-api.yml

```diff
@@ -113,6 +113,65 @@ paths:
           description: invalid input!
         "500":
           description: error!
+  /presentations/available:
+    post:
+      tags:
+        - Holder
+      summary: Notifies a holder of an available presentation.
+      operationId: notifyPresentationAvailable
+      description:
+        This API is the first part of a holder to holder credential exchange over http.
+        The server MAY reply with a query for the client to proceed with a presentation.
```

Comment:
Attempt at clearer description:
"Notifies this holder that the caller is prepared to deliver a presentation compliant with the provided Verifiable Presentation Request query. This is the first part of a presentation exchange from over this HTTP API. This holder (server) MUST return the required 'domain' and 'challenge' parameters for the Presentation. This holder (server) MAY reply with an alternate Verifiable Presentation Request query if it requires something different than the one proposed by the caller. The caller MUST return a Presentation in accordance with the response parameters."

Reply:
@mavarley can you either convert this to an issue or a suggested change, that I can approve.

Reply:
```suggestion
        Notifies this server-holder that the client-holder (caller) is prepared to deliver a presentation compliant with the provided Verifiable Presentation Request query. This is the first part of a presentation exchange over this HTTP API. This server-holder MUST return the required 'domain' and 'challenge' parameters for the Presentation. This server-holder MAY reply with an alternate Verifiable Presentation Request query if it requires something different than the one proposed by the client-holder. The client-holder MUST return a Presentation in accordance with the response parameters.
```